src/Action/LoginAction.php

```diff
@@ -86,12 +93,13 @@ public function __construct(
         $this->templateRegistry = $templateRegistry;
         $this->tokenStorage = $tokenStorage;
         $this->session = $session;
+        $this->translator = $translator;
```

Comment:
```suggestion
        // NEXT_MAJOR: Remove this block.
        if (null === $translator) {
            @trigger_error(sprintf(
                'Not passing an instance of "%s" as argument 6 to "%s()" is deprecated since'
                .' sonata-project/user-bundle 4.x and will be not possible in version 5.0.',
                TranslatorInterface::class,
                __METHOD__
            ), E_USER_DEPRECATED);
        }

        $this->translator = $translator;
```